Automattic/wp-calypso
# Marketplace: Stop using the redirectTo parameter and mount the Marketplace Thank you page URL with the proper slugs on the Checkout page

#### Proposed Changes

* Stop pass the `redirectTo` parameter
* Add the `plugin_slug` as extra property on the product - D97857-code
* Mount the Marketplace Thank you page URL with the slugs if a marketplace product is found on the cart

<img width="1499" alt="Screen Shot 2023-01-12 at 11 05 07" src="https://user-images.githubusercontent.com/5039531/212102273-a626733e-5ed7-4be0-824a-7d7c95ea55d8.png">

#### Testing Instructions
* Use the D97857-code change on your sandbox
* From a business or higher plan, go to a paid product page(`/plugins/{slug}/site`) and click on `Purchase and activate`
* Close the cart with the `X` on the top left of the screen and click on `Leave items`
* Go to another paid product page and click on `Purchase and activate`
* Complete the purchase
* You should see the Thank you page with all the purchased plugins



* From a below-than-business plan, repeat the steps above and you should see the same result.


#### Pre-merge Checklist





Fixes #71675
